doc/source/ray-air/getting-started.rst

```diff
@@ -44,31 +44,44 @@ Trainer
 .. autoclass:: ray.ml.trainer.Trainer
     :members:
 
-.. automodule:: ray.ml.train.integrations.xgboost
+
+Predictors
+~~~~~~~~~~
+
+.. autoclass:: ray.ml.predictor.Predictor
+    :members:
+
+.. autoclass:: ray.ml.batch_predictor.BatchPredictor
+    :members:
+
+Integrations
+~~~~~~~~~~~~
+
+.. automodule:: ray.ml.integrations.xgboost
     :members:
     :show-inheritance:
 
-.. automodule:: ray.ml.train.integrations.lightgbm
+.. automodule:: ray.ml.integrations.lightgbm
     :members:
     :show-inheritance:
 
-.. automodule:: ray.ml.train.integrations.tensorflow
+.. automodule:: ray.ml.integrations.tensorflow
     :members:
     :show-inheritance:
 
-.. automodule:: ray.ml.train.integrations.torch
+.. automodule:: ray.ml.integrations.torch
     :members:
     :show-inheritance:
 
-.. automodule:: ray.ml.train.integrations.sklearn
+.. automodule:: ray.ml.integrations.sklearn
     :members:
     :show-inheritance:
 
-.. autoclass:: ray.ml.train.data_parallel_trainer.DataParallelTrainer
+.. autoclass:: ray.ml.integrations.data_parallel_trainer.DataParallelTrainer
     :members:
     :show-inheritance:
 
-.. autoclass:: ray.ml.train.gbdt_trainer.GBDTTrainer
+.. autoclass:: ray.ml.integrations.gbdt_trainer.GBDTTrainer
```

Comment:
Hmm are these considered integrations?

Reply:
GBDT - yes, they are the base class for xgboost and lightgbm, both integrations.
DP - less clear hear imo, but again base class for mostly integrations. I would currently vote to keep them there so they are easy to find. The alternative would be to have `ray.ml.base` or `ray.ml.base_trainers` or so. 
What do other people think?